tools/serverpod_cli/test/analyzer/models/stateful_analyzer/model_validation/field/default/uuid/field_uuid_default_model_test.dart

```diff
@@ -0,0 +1,267 @@
+                .validateAll();
+
+        expect(collector.errors, isEmpty);
+
+        var definition = definitions.first as ClassDefinition;
+        expect(definition.fields.last.defaultModelValue, 'random');
+      },
+    );
+
+    test(
+      'when the field is of type UUID and the defaultModel is set to a valid UUID string with single quotes, then the field should have a "default model" value',
+      () {
+        var models = [
+          ModelSourceBuilder().withYaml(
+            '''
+          class: Example
+          table: example
+          fields:
+            uuidType: UuidValue, defaultModel='550e8400-e29b-41d4-a716-446655440000'
+          ''',
+          ).build()
+        ];
+
+        var collector = CodeGenerationCollector();
+        var definitions =
+            StatefulAnalyzer(config, models, onErrorsCollector(collector))
+                .validateAll();
+
+        expect(collector.errors, isEmpty);
+
+        var definition = definitions.first as ClassDefinition;
+
+        expect(
+          definition.fields.last.defaultModelValue,
+          '\'550e8400-e29b-41d4-a716-446655440000\'',
+        );
+      },
+    );
+
+    test(
+      'when the field is of type UUID and the defaultModel is set to a valid UUID string with double quotes, then the field should have a "default model" value',
+      () {
+        var models = [
+          ModelSourceBuilder().withYaml(
+            '''
+          class: Example
+          table: example
+          fields:
+            uuidType: UuidValue, defaultModel="550e8400-e29b-41d4-a716-446655440000"
+          ''',
+          ).build()
+        ];
+
+        var collector = CodeGenerationCollector();
+        var definitions =
+            StatefulAnalyzer(config, models, onErrorsCollector(collector))
+                .validateAll();
+
+        expect(collector.errors, isEmpty);
+
+        var definition = definitions.first as ClassDefinition;
+
+        expect(
+          definition.fields.last.defaultModelValue,
+          '"550e8400-e29b-41d4-a716-446655440000"',
+        );
+      },
+    );
+
+    test(
+      'when the field is of type UUID and the defaultModel is empty, then an error is generated',
+      () {
+        var models = [
+          ModelSourceBuilder().withYaml(
+            '''
+          class: Example
+          table: example
+          fields:
+            uuidType: UuidValue, defaultModel=
+          ''',
+          ).build()
+        ];
+
+        var collector = CodeGenerationCollector();
+        StatefulAnalyzer(config, models, onErrorsCollector(collector))
+            .validateAll();
+
+        expect(collector.errors, isNotEmpty);
+
+        var firstError = collector.errors.first as SourceSpanSeverityException;
+        expect(
+          firstError.message,
+          'The "defaultModel" value must be a valid UUID string or "random" (e.g., "defaultModel"=random or "defaultModel"="550e8400-e29b-41d4-a716-446655440000").',
```

Comment:
Suggestion: Flip the order of the examples so that it matches the order of the hint.

Reply:
Same for the other hints.